### ISSUE TITLE
Ensure that output from align sequence is displayed as stacked line plots

### ISSUE THREAD

**Issue by cmeyer** [nion-software/nionswift#1087]

**Comment by jamesrussell216** [comment]
@Tiomat85 can you put your explanation you just gave me of this in here please?!

**Comment by cmeyer** [comment]
I updated the main description of the bug. Is that sufficient?

**Comment by Tiomat85** [comment]
What Chris has put there is a far better description than you would have got from me.  I like the idea of a display hint going along with the data, as somewhere in the software should have some context as to what the data actually represents.

**Comment by jamesrussell216** [comment]
Yes thank you both - that's a big help, I actually understand the problem now!

Picking up on what @Tiomat85 said @cmeyer - we have that in the metadata somewhere at the moment? Like a property that is data item "type" or are most things just inferred of dims?

**Comment by jamesrussell216** [comment]
@cmeyer nudge

**Comment by jamesrussell216** [comment]
@cmeyer nudge
